components/builder-web/app/origin/origin-page/origin-page.component.ts

```diff
@@ -81,7 +77,7 @@ export class OriginPageComponent implements OnInit, OnDestroy {
     return this.store.getState().origins.mine;
   }
 
-  get iAmPartOfThisOrigin() {
```

Comment:
Part of me is sad at the departure of whimsical method names, but it shouldn't prevent merging.  😝

Reply:
Only to make it consistent with the others. I did shed a tear also when I changed it, though.